### PR TITLE
fix rtt_now when sim clock is running on xenomai

### PR DESCRIPTION
**Pull request by ahoarau**
Discussed here : 
https://github.com/jhu-lcsr/rtt_gazebo/issues/21

### CODE REVIEW

File: rtt_rosclock/src/rtt_rosclock.cpp [hunk 1]
```diff
@@ -55,13 +55,17 @@ const ros::Time rtt_rosclock::rtt_now()
   const uint64_t one_E9 = 1000000000ULL;
   // NOTE: getNSecs returns wall time, getTicks returns offset time
 #ifdef __XENO__
```
Comment (meyerj) [inline rtt_rosclock/src/rtt_rosclock.cpp:57]: Why is this distinction between Xenomai and non-Xenomai builds required here at all, independent of sim time?
Wouldn't `getTicks()` (the else branch) work for both cases?

`getTicks()` => `rt_timer_tsc()` (if system clock is enabled in the TimeService)
`getNSecs()` => `rt_timer_ticks2ns(rt_timer_read())` (independent of sim/system time mode)

Comment (ahoarau) [inline rtt_rosclock/src/rtt_rosclock.cpp:57]: I'll look into that next week. Thanks for the clarification !

Comment (ahoarau) [inline rtt_rosclock/src/rtt_rosclock.cpp:57]: That's very true, I'm closing this PR until I provide a new one with your recommendations :)

**Comment by ahoarau** [comment]
fixed PR with https://github.com/orocos/rtt_ros_integration/pull/65

**Pull request closed without merging**